doc/memos/rdm3.md

```diff
@@ -0,0 +1,103 @@
+RIOT Maintainers Community
+Developer Memo: 3
+Author(s): Martine Lenders
+Date: January 2017
```

Comment:
In rdm0 it parses correctly. Most likely because I "properly" set a for this document (which I'm not really happy with to revert)... I will try to find alternatives